lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -60,11 +60,16 @@
   }
 
   .tagline {
+    width: 75%;
```

Comment:
Confirmed its for everything, updated at the block level now rather than tagline